app/views/presentations/show.html.haml

```diff
@@ -0,0 +1,15 @@
+.content
+  %h1 #{@presentation.meeting.name}
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`